common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/StorageRandomThings.scala

```diff
@@ -185,6 +185,9 @@ trait StorageRandomThings extends RandomThings {
 
   def createBagPath: BagPath = BagPath(randomAlphanumeric)
 
+  def createBagPathWithPrefix(prefix: String, name: String): BagPath =
+    BagPath(f"$prefix/$name")
```

Comment:
nit: we tend to use s-formatted strings elsewhere. Is there a reason you've started to switch to f-strings?

Reply:
🐍 !